src/main/java/core/basesyntax/db/Storage.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax.db;
+
+import core.basesyntax.model.Bet;
+import core.basesyntax.model.Car;
+import java.util.ArrayList;
+import java.util.List;
+
+public class Storage {
+    public static final List<Bet> BETS_STORAGE = new ArrayList<>();
+    public static final List<Car> CARS_STORAGE = new ArrayList<>();
```

Comment:
these are not constants although they look similar

the names should be written in `camelCase`